### DISCUSSION TITLE
custom keybinds

### DISCUSSION THREAD

**Discussion by dacixn** [KadeArchive/Kade-Engine#692] [Ideas]
when are custom keybinds coming to standalone kade engine? (like from tricky v2)

**Comment by isakube** [comment]
happened

**Reply by dacixn** [reply]
my reaction as a clickbait title:

I predicted the next Friday Night Funkin' Update... (REAL) [NOT CLICKBAIT]

**Reply by isakube** [reply]
Do not.

**Reply by PandaFNF** [reply]
they are here in a new update
